lib/freebsd.ex

```diff
@@ -5,17 +5,24 @@ defmodule FreeBSD do
   """
 
   def pkg_manifest do
+    pkg_name = pkg_name()
+
     %{
-      name: pkg_name(),
+      name: pkg_name,
       version: pkg_version(),
-      origin: "devel/#{pkg_name()}",
+      origin: "devel/#{pkg_name}",
       comment: pkg_comment(),
       www: pkg_www(),
       maintainer: pkg_maintainer(),
       prefix: pkg_prefix(),
-      desc: pkg_description()
+      desc: pkg_description(),
+      scripts: %{
+        "post-install" => "/usr/local/libexec/#{pkg_name}/bin/post-install",
```

Comment:
`/usr/local` should not be hard-coded, and instead should be `pkg_prefix()`. _That_ is currently hard-coded to `/usr/local`, but the idea is that you can install to any prefix you want (this is a FreeBSD thing), and files should be installed relative to that prefix.